src/RewardsDistributor.sol

```diff
@@ -53,7 +55,13 @@ contract RewardsDistributor is IRewardDistributor {
         if (msg.sender != rewardManager) {
             revert AccessError.Unauthorized(msg.sender);
         }
-        IERC20(collateralType).transfer(payoutTarget_, payoutAmount_);
+        if (poolId_ != poolId) {
+            revert ParameterError.InvalidParameter("poolId", "Unexpected pool");
```

Comment:
Nit but might be better to say why it was unexpected (ie doesn't match internal records/disallowed)

Reply:
Sure. Couldn't find the right wording tho. Will think tomorrow